api/config/packages/dev/doctrine.yaml

```
@@ -0,0 +1,4 @@
+doctrine:
+  dbal:
+    # "TEST_TOKEN" is typically set by ParaTest
+    dbname_suffix: 'dev%env(default::TEST_TOKEN)%'
```

i don't think you need the test token for dev

---

This is needed

---

you need to add the suffix, but you don't need a test token.
ParaTest will never set the TEST_TOKEN for the dev environment, only in the test environment.

```suggestion
    dbname_suffix: 'dev'
```